features/networking/sdn.feature

```diff
@@ -258,3 +258,42 @@ Feature: SDN related networking scenarios
     When I run cmds on all ovs pods:
       | bash | -c | pgrep ovs-vswitchd |
     Then the step should succeed
+
+  #@author huirwang@redhat.com
```

Comment:
Please check the format others are using. Some of auto tools rely on that.

Reply:
Updated, thanks!